src/libraries/System.Runtime/tests/System.IO.FileSystem.Tests/DirectoryInfo/CreateSubdirectory.cs

```diff
@@ -235,6 +236,117 @@ public void ParentDirectoryNameAsPrefixShouldThrow()
             Assert.Throws<ArgumentException>(() => di.CreateSubdirectory(Path.Combine("..", randomName + "abc", GetTestFileName())));
         }
 
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsWindows), nameof(PlatformDetection.IsSubstAvailable))]
```

Comment:
Use `[PlatformSpecific(TestPlatforms.Windows)]` instead of `PlatformDetection.IsWindows`.